.github/workflows/unittests.yml

```diff
@@ -28,5 +28,8 @@ jobs:
     - name: Install Nodejs dependencies
       run: |
         npm ci
+    - name: Run Deployment Script
+      run: |
+        ./build-tools/heroku/on-deploy.sh
```

Comment:
Do we need the whole script? For the bug we are addressing, maybe `python ../../content/yaml_to_lark_utils.py` is enough? (We can try to see if that fixes the e2e test, even though I am also curious why that happens)

Reply:
That solved the problem indeed!